src/screens/ExplainSheet.js

```diff
@@ -33,68 +42,125 @@ const VERIFIED_EXPLAINER = `Tokens with a verified badge mean they have appeared
 
 Always do your own research to ensure you are interacting with a token you trust.`;
 
-const explainers = {
+const OPTIMISM_EXPLAINER = `Optimism is a Layer 2 network that runs on top of Ethereum, enabling cheaper and faster transactions while still benefiting from the underlying security of Layer 1.
+
+It bundles lots of transactions together in a "roll up" before sending them down to live permanently on Layer 1.`;
+
+const ARBITRUM_EXPLAINER = `Arbitrum is a Layer 2 network that runs on top of Ethereum, enabling cheaper and faster transactions while still benefiting from the underlying security of Layer 1.
+
+It bundles lots of transactions together in a "roll up" before sending them down to live permanently on Layer 1.`;
+
+const POLYGON_EXPLAINER = `Polygon is a sidechain, a distinct network that runs alongside Ethereum and is compatible with it. 
+
+It allows for cheaper and faster transactions, but unlike Layer 2 networks, Polygon has its own security and consensus mechanisms that differ from Ethereum.`;
+
+export const explainers = {
   gas: {
     emoji: '⛽️',
     text: GAS_EXPLAINER,
     title: 'Ethereum network fee',
   },
+  sending_funds_to_contract: {
+    emoji: '✋',
+    text: SENDING_FUNDS_TO_CONTRACT,
+    title: 'Hold your horses!',
+    extraHeight: 80,
+  },
   verified: {
     emoji: '􀇻',
     text: VERIFIED_EXPLAINER,
     title: 'Verified Tokens',
   },
+  optimism: {
+    emoji: '⛽️',
+    logo: (
+      <ChainLogo marginBottom={10} network={networkTypes.optimism} size={100} />
+    ),
+    text: OPTIMISM_EXPLAINER,
+    title: `What's Optimism?`,
+    readMoreLink: 'https://optimism.io/',
+    extraHeight: 120,
+  },
+  arbitrum: {
+    emoji: '⛽️',
+    logo: (
+      <ChainLogo marginBottom={10} network={networkTypes.arbitrum} size={100} />
+    ),
+    text: ARBITRUM_EXPLAINER,
+    title: `What's Arbitrum?`,
+    readMoreLink: 'https://arbitrum.io/',
+    extraHeight: 120,
+  },
+  polygon: {
+    emoji: '⛽️',
+    logo: (
+      <ChainLogo marginBottom={10} network={networkTypes.polygon} size={100} />
+    ),
+    text: POLYGON_EXPLAINER,
+    title: `What's Polygon?`,
+    readMoreLink: 'https://polygon.technology/',
+    extraHeight: 120,
+  },
 };
 
 const SavingsSheet = () => {
```

Comment:
code cleanup from previous changes: can we rename `SavingsSheet` here to `ExplainSheet`